testdata/cheats/Cheats.sol

```diff
@@ -6,6 +6,8 @@ interface Cheats {
     struct Log {bytes32[] topics; bytes data;}
     // Set block.timestamp (newTimestamp)
     function warp(uint256) external;
+    // Set block.difficulty (newDifficulty)
+    function whelm(uint256) external;
```

Comment:
I'd like to suggest a more intuitive name 😅
```suggestion
    function difficulty(uint256) external;
```

Reply:
Changed